packages/debug/src/browser/debug-session.tsx

```diff
@@ -419,8 +419,16 @@ export class DebugSession implements CompositeTreeElement {
             const thread = existing.get(id) || new DebugThread(this);
             this._threads.set(id, thread);
             const data: Partial<Mutable<DebugThreadData>> = { raw };
-            if (stoppedDetails && (stoppedDetails.allThreadsStopped || stoppedDetails.threadId === id)) {
-                data.stoppedDetails = stoppedDetails;
+            if (stoppedDetails) {
+                if (stoppedDetails.threadId === id) {
+                    data.stoppedDetails = stoppedDetails;
+                } else if (stoppedDetails.allThreadsStopped) {
+                    data.stoppedDetails = { ...stoppedDetails };
```

Comment:
or `data.stoppedDetails = { reason: '' }`? I don't see that we read anything else, also I'm not sure that we should read other properties or they are relevant only to `threadId`

Reply:
Yes, it is simple, let's go with that for now.